crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -618,6 +618,59 @@ where
         self.add_list_color(polymorphic_color, 6);
     }
 
+    /// Draw a wireframe cuboid in 3D.
+    ///
+    /// This should be called for each frame the cuboid needs to be rendered.
+    ///
+    /// # Example
+    /// ```
+    /// # use bevy_gizmos::prelude::*;
+    /// # use bevy_transform::prelude::*;
+    /// # use bevy_math::{bounding::Aabb3d, Vec3};
+    /// # use bevy_color::palettes::basic::GREEN;
+    /// fn system(mut gizmos: Gizmos) {
+    ///     gizmos.cuboid(Aabb3d::new(Vec3::ZERO, Vec3::ONE), Transform::IDENTITY, GREEN);
+    /// }
+    /// # bevy_ecs::system::assert_is_system(system);
+    /// ```
+    #[inline]
+    pub fn cuboid(
+        &mut self,
+        aabb: impl Into<Aabb3d>,
```

Comment:
I don't love that this is called a `cuboid` but it then takes an `Aabb3d`. I would expect `cuboid` to take a `Cuboid`. For an AABB method, I'd just call this `aabb_3d`

Reply:
Does `Aabb3d` have a `From<Cuboid>`? Alternatively can we make this `impl Into<Cuboid>`?

Reply:
An AABB does have a cuboid shape, but it also has the added semantics of being a bounding box for some other object or shape. It's not *just* some abstract cuboid. That's part of why it doesn't make sense to me to take an `Aabb3d` in a method called `cuboid`, in addition to the general weirdness of having the names of the method and input type not match.

The method name here is also just not what I'd search for. If I want to draw an `Aabb3d`, I would search for some `aabb_` method, not `cuboid`. So yeah, I would much prefer an explicit `aabb_3d` (and `aabb_2d`) method